### PR TITLE
update ocamltop-gtk opam files; require ocaml < 4.03.0

### PR DESCRIPTION
**Pull request by zoggy**

### CODE REVIEW

File: packages/ocamltop-gtk/ocamltop-gtk.2.1.1/opam [hunk 1]
```diff
@@ -1,13 +1,25 @@
...
+
 build: [
   ["./configure" "--prefix" prefix]
   [make "all"]
   [make "install-lib"]
 ]
-remove: [[make "uninstall-lib"]]
+remove: [["ocamlfind" "remove" "ocamltop-gtk"]]
 depends: [
   "ocamlfind"
```
Comment (dsheets) [inline packages/ocamltop-gtk/ocamltop-gtk.2.1.1/opam]: `{build}`?

File: packages/ocamltop-gtk/ocamltop-gtk.2.1.1/opam [hunk 2]
```diff
@@ -1,13 +1,25 @@
-opam-version: "1"
+opam-version: "1.2"
 maintainer: "zoggy@bat8.org"
+authors: ["Maxence Guesdon"]
+
+homepage: "http://zoggy.github.io/ocamltop-gtk/"
+license: "GNU Lesser General Public License version 3"
```
Comment (dsheets) [inline packages/ocamltop-gtk/ocamltop-gtk.2.1.1/opam]: `LGPL-3.0` is the SPDX name. Could you please use this name?

**Comment by yallop** [comment]
The build's [failing](https://travis-ci.org/ocaml/opam-repository/jobs/127501806) on 4.02:

```
### stderr ###
# findlib: [WARNING] Interface topdirs.cmi occurs in several directories: /usr/lib/ocaml, /usr/lib/ocaml/compiler-libs
# findlib: [WARNING] Interface topdirs.cmi occurs in several directories: /usr/lib/ocaml, /usr/lib/ocaml/compiler-libs
# findlib: [WARNING] Interface topdirs.cmi occurs in several directories: /usr/lib/ocaml, /usr/lib/ocaml/compiler-libs
# findlib: [WARNING] Interface topdirs.cmi occurs in several directories: /usr/lib/ocaml, /usr/lib/ocaml/compiler-libs
# File "ocamltop.ml", line 93, characters 23-28:
# Error: This pattern matches values of type 'a * 'b
#        but a pattern was expected which matches values of type
#          Parsetree.extension_constructor
# make: *** [ocamltop.cmo] Error 2
```

**Comment by zoggy** [comment]
Let's remove this package, since it seems unused and I don't intend to maintain it. Better alternatives exist with utop and ocaml-top.

**Comment by yallop** [comment]
Ok; since nothing depends on this package, removing it is unlikely to cause problems.

**Pull request merged**